wgpu-core/src/validation.rs

```diff
@@ -20,6 +20,33 @@ enum ResourceType {
     AccelerationStructure,
 }
 
+#[derive(Clone, Debug)]
+pub enum BindingTypeName {
+    Buffer,
+    Texture,
+    Sampler,
+    AccelerationStructure,
+}
+
+fn map_resource_to_type_name(ty: &ResourceType) -> BindingTypeName {
```

Comment:
I would expect this to be an inherent function on BindingTypeName, like `BindingTypeName::from_resource_type`

Reply:
Actually I think an actual `From` impl will probably be best, then the call sites can just use `into`.

Reply:
Will do! I admit I was really not sure where to put this, not 100% familiar with rust idioms, thanks for the feedback. :)